<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Huffman</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>85</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCompress">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter with file to compress or descompress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NavCompress_bt">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nav_txt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>300</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Decopress_bt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEnter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Compress_bt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="NameFile">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>121</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Type">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>140</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="Size">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>159</y>
      <width>19</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameResult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TypeResult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SizeResult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
